ethcore/src/engines/block_reward.rs

```diff
@@ -14,13 +14,17 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+//! A module with types for declaring block rewards and a client interface for interacting with a
```

Comment:
I'm not sure `engines` is the appropriate module for `block_reward`. Any thoughts?